frontend_app_caldavzap/webpack.config.js

```diff
@@ -0,0 +1,91 @@
+const path = require('path')
+const isProduction = process.env.NODE_ENV === 'production'
+
+console.log('isProduction : ', isProduction)
+
+module.exports = {
+  mode: isProduction ? 'production' : 'development',
+  entry: isProduction
+    ? './src/index.js' // only one instance of babel-polyfill is allowed
+    : ['@babel/polyfill', './src/index.dev.js'],
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: isProduction ? 'caldavzap.app.js' : 'caldavzap.app.dev.js',
+    pathinfo: !isProduction,
+    library: isProduction ? 'appCaldavzap' : undefined,
+    libraryTarget: isProduction ? 'var' : undefined
+  },
+  externals: {},
+  // isProduction ? { // Côme - since plugins are imported through <script>, cannot externalize libraries
+  //   react: {commonjs: 'react', commonjs2: 'react', amd: 'react', root: '_'},
+  //   'react-dom': {commonjs: 'react-dom', commonjs2: 'react-dom', amd: 'react-dom', root: '_'},
+  //   classnames: {commonjs: 'classnames', commonjs2: 'classnames', amd: 'classnames', root: '_'},
+  //   'prop-types': {commonjs: 'prop-types', commonjs2: 'prop-types', amd: 'prop-types', root: '_'},
+  //   tracim_frontend_lib: {commonjs: 'tracim_frontend_lib', commonjs2: 'tracim_frontend_lib', amd: 'tracim_frontend_lib', root: '_'}
+  // }
+  // : {},
```

Comment:
INFO - 2019-03-29 - CH - since plugins are imported through <script>, cannot externalize libraries

Reply:
to remove

Reply:
comment removed